packages/lib/src/spatial-navigation/SpatialNavigator.ts

```diff
@@ -1,11 +1,22 @@
 import { Direction, Lrud } from '@bam.tech/lrud';
 
+export type OnDirectionHandledWithMovement = (direction: Direction) => void;
+type OnDirectionHandledWithMovementRef = { current: OnDirectionHandledWithMovement };
+
+type SpatialNavigatorParams = {
+  onDirectionHandledWithoutMovementRef: OnDirectionHandledWithMovementRef;
+};
```

Comment:
Is this a naming error OnDirectionHandled**With**MovementRef ?

Reply:
indeed, this is a mistake! thanks!